server/src/services/apiKey.js

```diff
@@ -0,0 +1,23 @@
+const { NotFoundError } = require('../utils/errors')
```

Comment:
Looks like this file was for a different PR

Reply:
Woops...